Gemfile

```
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby IO.read(File.expand_path('.ruby-version', __dir__)).strip
+ruby File.read('.ruby-version').strip
```

I still want the "in the same directory as the `Gemfile`" part of this:

```suggestion
ruby File.read(File.expand_path('.ruby-version', __dir__)).strip
```
---

```bash
$ cd app
$ bundle


[!] There was an error parsing `Gemfile`: No such file or directory @ rb_sysopen - .ruby-version. Bundler cannot continue.

 #  from /home/matt/projects/screaming-dinosaur/Gemfile:4
 #  -------------------------------------------
 #  source 'https://rubygems.org'
 >  ruby File.read('.ruby-version').strip
 #  
 #  -------------------------------------------
```